### DISCUSSION TITLE
Use manifest exclusively on localhost to build static site with Svelte?

### DISCUSSION THREAD

**Discussion by eballeste** [mnfst/llm-gateway#221] [Feature request]
I am planning out a very small portfolio-like project and want to learn Svelte while using it as a static site generator. 

I was thinking of using MD files to keep some sort of structure for the site's content but I just happened to find out about manifest.js as a backend service and now I'm wondering if I would be able to use manifest, exclusively on my localhost, in order to feed the Svelte build process to generate static files??? Does this sound far-fetched?

Has anybody done something like this?

**Comment by brunobuddy** [comment]
@eballeste Yes you can do that with SvelteKit using the `+page` syntax with the `prerender = true` option (SSG). Do you want me to make an example repo ?

**Reply by eballeste** [reply]
No worries!! 

I just wanted to know if this was feasible or if I was overthinking this, I'll try to figure it out, thank you so much!

**Comment by brunobuddy** [comment]
Closing this out. It's from Manifest's old backend-as-a-service days, which we don't work on anymore since the pivot. See #742 for where things are now.
